documentation/decisions/008_support_theming.md

```diff
@@ -0,0 +1,244 @@
+---
```

Comment:
Callout that this file name may need to change depending on the order you merge!